### PR TITLE
Feature/section 9 subscribe to our newsletter 

### PR DESCRIPTION
**Pull request by Dilekcn**

### CODE REVIEW

**Review by youssef910 — APPROVED** [review]

File: client/src/pages/Home/SubscribeNews/SubscribeNews.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+					<button  onClick ={alertMess} className="subscribeNews_button">
+						<FaChevronRight />
+        	</button>
+				</div>
+			</div>
+
+			<div className="subscribeNews_img_container">
+				<div className="subscribeNews_img">
+					<img
+						src="https://i.postimg.cc/65LfHnkV/woman-using-a-mobile-phone-in-the-kitchen-2021-08-30-13-43-41-utc-1.jpg"
```
Comment (filkovsp) [inline client/src/pages/Home/SubscribeNews/SubscribeNews.js]: Hi Dilek
please place this image into local file.
for example:
`client\src\pages\Home\images\woman-using-a-mobile-phone-in-the-kitchen,jpg`
and link it from there.
please remove unnecessary symbols and letters in image file name.

**Review by filkovsp — CHANGES_REQUESTED** [review]

File: client/src/pages/Home/SubscribeNews/SubscribeNews.js (hunk 1 above)
Comment (Dilekcn) [reply client/src/pages/Home/SubscribeNews/SubscribeNews.js]: Hi @filkovsp  thank you for your commit.
Unnecessary symbols removed and letters in the image file name.
 This image is placed into a local file.

Comment (Tandy-Nicole) [reply client/src/pages/Home/SubscribeNews/SubscribeNews.js]: @filkovsp has the conversation been resolved? Before I accept Approval and merge PR.

**Review by youssef910 — APPROVED** [review]

File: client/src/pages/Home/SubscribeNews/SubscribeNews.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+						Subscribe to <br /> our newsletter
+					</h2>
+				</div>
+				<div className="subscribeNews_form">
+					<input
+						className="subscribeNews_input"
+						type="text"
+						placeholder="What is your email address"
+					/>
+					<button onClick={alertMess} className="subscribeNews_button">
```
Comment (youssef910) [inline client/src/pages/Home/SubscribeNews/SubscribeNews.js:25]: the file need proper indentation

**Review by youssef910 — CHANGES_REQUESTED** [review]

**Review by filkovsp — CHANGES_REQUESTED** [review]
Hi Dilek, 
everything is basically fine. 
please let me ask you to do just few minor cosmetic changes,
sorry, for pouring then all-in-one on to you :)

* in `client\src\pages\Home\AboutUs\AboutUs.js`
you use 
src/assets/images/about-us-concept-2021-08-31-16-59-05-utc.jpg
rename it to about-us-concept.jpg

* remove unused file:
client\src\image\about-us-concept-2021-08-31-16-59-05-utc.jpg

* remove unused file:
client\src\image\autistic-little-girl-close-up-2021-09-24-04-06-23-utc.jpg

* remove unused file:
client\src\image\cute-girl-with-down-syndrome-in-mothers-arms-2021-09-24-04-06-20-utc.jpg

* remove unused file:
client\src\pages\logo.svg

* remove unused file:
client\src\image\woman.jpg

* remove empty folder:
client\src\pages\Home\images

* fix MemberSays section:
in file client\src\pages\Home\MemberSays\Member.css
set height for .memberSays-container:

.memberSays-container {
	fit-content;
}

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]

**Review by filkovsp — APPROVED** [review]

**Pull request merged**